accounting/filters/OsgScheddCpuRemovedFilter.py

```diff
@@ -99,7 +99,9 @@ def schedd_filter(self, data, doc):
         # Get output dict for this schedd
         schedd = i.get("ScheddName", "UNKNOWN") or "UNKNOWN"
         o = data["Schedds"][schedd]
-
+       
+        if i.get("JobUniverse",5)==12:
+            return
```

Comment:
I don't understand the need to filter out all local universe jobs from the report?